machineSetup/setupPerfMachine.ps1

```diff
@@ -0,0 +1,178 @@
+﻿#Set power scheme to high performance
+powercfg /S 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_SLEEP STANDBYIDLE 0
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_VIDEO VIDEOIDLE 0
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR PERFBOOSTMODE 0
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR PROCTHROTTLEMAX 100
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR PROCTHROTTLEMIN 100
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR SYSCOOLPOL 1  
+powercfg /SETACVALUEINDEX SCHEME_MIN SUB_PROCESSOR THROTTLING 0  
+#region GIT Install Config
+$gitInstallLog = @"
+[Setup]
+Lang=default
+Dir=C:\Program Files\Git
+Group=Git
+NoIcons=0
+SetupType=default
+Components=ext,ext\shellhere,ext\guihere,assoc,assoc_sh
+Tasks=
+PathOption=Cmd
+SSHOption=OpenSSH
+CRLFOption=CRLFAlways
+BashTerminalOption=ConHost
+PerformanceTweaksFSCache=Enabled
+UseCredentialManager=Enabled
+EnableSymlinks=Disabled
+"@
+#endregion
+$gitInstallLog > gitInstallInfo.log
+wget https://github.com/git-for-windows/git/releases/download/v2.14.1.windows.1/Git-2.14.1-64-bit.exe -OutFile git-2.14.1.exe
+$gitArgList = "/LOADINF=gitInstallInfo.log", "/VERYSILENT"
+echo Start-Process -FilePath .\git-2.14.1.exe -ArgumentList $gitArgList -Wait
+Start-Process -FilePath .\git-2.14.1.exe -ArgumentList $gitArgList -Wait
+
+wget https://cmake.org/files/v3.7/cmake-3.7.2-win64-x64.msi -OutFile cmake-3.7.2.msi
+$cmakeArgList = "/i", "cmake-3.7.2.msi", "/quiet", "/norestart", "ADD_CMAKE_TO_PATH=System"
+echo Start-Process -FilePath msiexec -ArgumentList $cmakeArgList -Wait
+Start-Process -FilePath msiexec -ArgumentList $cmakeArgList -Wait
+
+wget http://javadl.oracle.com/webapps/download/AutoDL?BundleId=218831_e9e7ea248e2c4826b92b3f075a80e441 -OutFile java.exe
+$javaArgList = ,"/s"
+echo Start-Process -FilePath java.exe -ArgumentList $javaArgList -Wait
+Start-Process -FilePath java.exe -ArgumentList $javaArgList -Wait
+
+wget https://www.python.org/ftp/python/2.7.13/python-2.7.13.msi -OutFile python27.msi
+$python27ArgList = "/i", "python27.msi", "/quiet", "/norestart", "ADDLOCAL=DefaultFeature,SharedCRT,Extensions,TclTk,Documentation,Tools,pip_feature,Testsuite,PrependPath"
+echo Start-Process -FilePath msiexec -ArgumentList $python27ArgList -Wait
+Start-Process -FilePath msiexec -ArgumentList $python27ArgList -Wait
+
+wget https://www.python.org/ftp/python/3.5.3/python-3.5.3.exe -OutFile python35.exe
+$python35ArgList = "/quiet", "PrependPath=1", "TargetDir=C:\Python35"
+echo Start-Process -FilePath python35.exe -ArgumentList $python35ArgList -Wait
+Start-Process -FilePath python35.exe -ArgumentList $python35ArgList -Wait
+
+wget https://benchviewstorage.blob.core.windows.net/public/vs_community.exe -OutFile vs_community.exe
```

Comment:
We should use this Url: `https://download.microsoft.com/download/0/B/C/0BC321A4-013F-479C-84E6-4A2F90B11269/vs_community.exe` directly from Microsoft, or move to choco. The BenchView storage does not exist anymore.